Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+
+  <PropertyGroup>
+    <Version>1.0.0-alpha.1</Version>
+    <Authors>Ulrich Telle</Authors>
```

Comment:
Well, I would propose that you add your name at least here as it would be a bit arrogant to claim that I am the sole author.